packages/augur-sdk/src/state/db/MarketDB.ts

```diff
@@ -59,26 +59,26 @@ export class MarketDB extends DerivedDB {
       },
     };
 
-    const result = await this.stateDB.findInDerivedDB(this.stateDB.getDatabaseName("CurrentOrders"), request);
+    const result = await this.stateDB.findInDerivedDB(this.stateDB.getDatabaseName('CurrentOrders'), request);
```

Comment:
nit pic, shouldn't database names have constants for look ups?

Reply:
True, this can be changed to `DB.findCurrentOrderLogs`. Will update.